docs/how_to/create_custom_coords.rst

```diff
@@ -0,0 +1,36 @@
+.. _sunpy-how-to-create-custom-coordinate-objects:
+
+**********************************************
+Create Coordinate Objects with Custom Observer
+**********************************************
+
+It is possible to specify an observer other than Earth by providing a `~astropy.coordinates.SkyCoord` object representing the observer's location. This can be useful when working with data from instruments located at specific points in space, such as spacecraft.
```

Comment:
```suggestion
It is possible to specify an observer other than Earth by providing a `~astropy.coordinates.SkyCoord` object representing the observer's location.
This can be useful when working with data from instruments located at specific points in space, such as spacecraft.
```